docs/faq.rst

```diff
@@ -47,8 +47,6 @@ Due to these complexities, we decided not to support this functionality.
 
 However, it’s possible to register and use these security keys when passwordless login functionality is disabled. When used as a :term:`second-factor authentication`, the user is already partially authenticated so it’s much more straightforward to return credential IDs.
 
-If you are curious, the option used to restrict registration is done by setting the ``residentKey`` :term:`authenticator` selection criteria to required. See the specification for details: `authenticatorSelection.residentKey in the Web Authentication specification <https://www.w3.org/TR/webauthn-2/#dom-authenticatorselectioncriteria-residentkey>`_.
-
 You can still register and use these security keys when passwordless login is disabled. When used as a second-factor authentication method, the user is already partially authenticated, making it easier to return credential IDs securely.
 
-If you want more details, this restriction is done by setting the residentKey authenticator selection criteria to required. For more information, see `authenticatorSelection.residentKey in the Web Authentication specification <https://www.w3.org/TR/webauthn-2/#dom-authenticatorselectioncriteria-residentkey>`_.
+If you are curious, you can restrict registration by setting the ``residentKey`` :term:`authenticator` selection criteria to required. See the specification for details: `authenticatorSelection.residentKey in the Web Authentication specification <https://www.w3.org/TR/webauthn-2/#dom-authenticatorselectioncriteria-residentkey>`_.
```

Comment:
@activus-d please revert this. This isn't something 'you can do', it is already done -- as indicated in the original wording.